tox.ini

```diff
@@ -67,5 +67,4 @@ commands =
     rm tests/__init__.py
     pycodestyle enterprise tests test_utils
     pydocstyle enterprise tests test_utils
-    isort --check-only --recursive tests enterprise test_utils manage.py setup.py test_settings.py
```

Comment:
What specifically was the problem here? I think it would be better to disable or change the parameters of the pylint check if it conflicts with isort, rather than removing isort completely.